docroot/profiles/custom/sul_profile/modules/sul_helper/src/Plugin/paragraphs/Behavior/CardBehaviors.php

```diff
@@ -54,6 +54,34 @@ public function buildBehaviorForm(ParagraphInterface $paragraph, array &$form, F
         'cta_button' => $this->t('CTA'),
       ],
     ];
+    $element['background'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Background Color'),
+      '#description' => $this->t('Choose a background for the component.'),
+      '#empty_option' => $this->t('- Change the background -'),
+      '#default_value' => $paragraph->getBehaviorSetting('sul_teaser_styles', 'background'),
```

Comment:
This must match the ID of this plugin all the time. [see the id](https://github.com/SU-SWS/ace-sulgryphon/pull/29/files#diff-dd83ebd19befcb103b16eec970b6303b9e4443d7d58b88dba133e0cd7b173968R16)
```suggestion
      '#default_value' => $paragraph->getBehaviorSetting('sul_card_styles', 'background'),
```